### PR TITLE
[onert] Update TrainingInfo::isValid() condition

### PR DESCRIPTION
**Pull request by zetwhite**
This PR update TrainingInfo::isValid() condtion to check learning_rate has positive value.

ONE-DCO-1.0-Signed-off-by: SeungHui Youn <sseung.youn@samsung.com>

### CODE REVIEW

File: runtime/onert/core/src/ir/train/TrainingInfo.cc [hunk 1]
```diff
@@ -31,7 +31,7 @@ bool TrainingInfo::isValid() const
   if (_optimizer_info.optim_code == onert::ir::train::OptimizerCode::Invalid)
     return false;
 
-  if (_optimizer_info.learning_rate == 0.0f)
+  if (_optimizer_info.learning_rate <= 0.0f)
```
Comment (zetwhite) [inline runtime/onert/core/src/ir/train/TrainingInfo.cc:34]: background : https://github.com/Samsung/ONE/pull/12340#discussion_r1432642877 

> (Optional)
> If learning rate is a negative value, outputs will move away from the minimum after training. In other words, outputs will move to reverse what gradient descent does, and nn model won't be trained at all.
So, it would be better to change the condition if this function is for checking valid info of `TrainingInfo`, not default values.

**Review by SlavikMIPT — APPROVED** [review]
LGTM

**Review by ragmani — APPROVED** [review]
LGTM

**Review by glistening — APPROVED** [review]
LGTM
